<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>761</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1227</width>
    <height>761</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>1233</width>
      <height>682</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="minimumSize">
        <size>
         <width>911</width>
         <height>650</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>WorkSpace</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>0</y>
           <width>911</width>
           <height>661</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="minimumSize">
          <size>
           <width>311</width>
           <height>301</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>311</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Entity</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>301</width>
             <height>271</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4"/>
          </widget>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Rotate</string>
          </attribute>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>240</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>281</width>
             <height>191</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>绕x轴</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider">
                <property name="minimum">
                 <number>-180</number>
                </property>
                <property name="maximum">
                 <number>180</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox">
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>绕y轴</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_2">
                <property name="minimum">
                 <number>-180</number>
                </property>
                <property name="maximum">
                 <number>180</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>绕z轴</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_3">
                <property name="minimum">
                 <number>-180</number>
                </property>
                <property name="maximum">
                 <number>180</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <zorder>layoutWidget</zorder>
          <zorder>pushButton</zorder>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="minimumSize">
          <size>
           <width>311</width>
           <height>370</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>311</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string notr="true">Object List</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionImport_OBJ_File"/>
    <addaction name="actionExport_OBJ_File"/>
    <addaction name="separator"/>
    <addaction name="actionScreen_Capture"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEnity">
    <property name="title">
     <string>Enity</string>
    </property>
    <addaction name="actionAdd_cube"/>
    <addaction name="actionAdd_prism"/>
    <addaction name="actionAdd_prismoid"/>
    <addaction name="actionAdd_sphere"/>
    <addaction name="actionAdd_cylinder"/>
    <addaction name="actionAdd_cone"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionLight_On_Off"/>
    <addaction name="actionWire_Solid"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEnity"/>
   <addaction name="menuControl"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave_Project"/>
   <addaction name="separator"/>
   <addaction name="actionImport_OBJ_File"/>
   <addaction name="actionExport_OBJ_File"/>
   <addaction name="separator"/>
   <addaction name="actionLight_On_Off"/>
   <addaction name="actionWire_Solid"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_Project">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/addProject.ico</normaloff>:/new/prefix1/Icons/addProject.ico</iconset>
   </property>
   <property name="text">
    <string>Create Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/openProject.ico</normaloff>:/new/prefix1/Icons/openProject.ico</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/saveProject.ico</normaloff>:/new/prefix1/Icons/saveProject.ico</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionImport_OBJ_File">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/openFile.ico</normaloff>:/new/prefix1/Icons/openFile.ico</iconset>
   </property>
   <property name="text">
    <string>Import OBJ File</string>
   </property>
  </action>
  <action name="actionExport_OBJ_File">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/outputFile.ico</normaloff>:/new/prefix1/Icons/outputFile.ico</iconset>
   </property>
   <property name="text">
    <string>Export OBJ File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_cube">
   <property name="text">
    <string>Add cube</string>
   </property>
  </action>
  <action name="actionLight_On_Off">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/lightOn.ico</normaloff>:/new/prefix1/Icons/lightOn.ico</iconset>
   </property>
   <property name="text">
    <string>Light On / Off</string>
   </property>
  </action>
  <action name="actionWire_Solid">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/wireMode.ico</normaloff>:/new/prefix1/Icons/wireMode.ico</iconset>
   </property>
   <property name="text">
    <string>Wire / Solid</string>
   </property>
  </action>
  <action name="actionAdd_prism">
   <property name="text">
    <string>Add prism</string>
   </property>
  </action>
  <action name="actionAdd_prismoid">
   <property name="text">
    <string>Add prismoid</string>
   </property>
  </action>
  <action name="actionAdd_sphere">
   <property name="text">
    <string>Add sphere</string>
   </property>
  </action>
  <action name="actionScreen_Capture">
   <property name="text">
    <string>Screen Capture</string>
   </property>
  </action>
  <action name="actionAdd_cylinder">
   <property name="text">
    <string>Add cylinder</string>
   </property>
  </action>
  <action name="actionAdd_cone">
   <property name="text">
    <string>Add cone</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>initSpinBoxAndSlider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1069</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initSpinBoxAndSlider()</slot>
  <slot>updateCatalog()</slot>
 </slots>
</ui>
